packages/lib-classifier/src/store/tasks/Graph2dRangeXTask.js

```diff
@@ -0,0 +1,13 @@
+import { types } from 'mobx-state-tree'
+import Task from './Task'
+
+const Graph2dRangeX = types.model('Graph2dRangeX', {
+  help: types.optional(types.string, ''),
+  instruction: types.string,
+  required: types.maybe(types.boolean), // Should this be an optional type with the default to true?
```

Comment:
It should probably be `required: types. optional(types.boolean, false)`, since there may not be anything on the subject to mark

Reply:
👌